inst/rmarkdown/templates/isba/skeleton/badoc.tex

```diff
@@ -0,0 +1,455 @@
+% \documentclass[ba,preprint]{imsart}% use this for supplement article
```

Comment:
I don't think this file should be included in the **rticles** package. 

README says : 
> -   `badoc.tex` - LaTeX source file of the sample paper. 
    It contains useful information about typseting an article.

We don't need that, do we ? 

At least, we should add in Rbuildignore if we just want to keep a copy in Git.

Reply:
Removed